app/models/registration/item.rb

```diff
@@ -0,0 +1,18 @@
+module Registration
+  class Item < ApplicationRecord
```

Comment:
The only reason why we differentiate between `Registration::Item` and `Registerable` is this:

> Users register for a `Registration::Item`, not directly for a `Registerable`. This separation allows the same Tutorial to potentially be part of different campaigns over time without conflict.

Do we want to allow a Tutorial to be in two different campaigns _at the same time_? If not, then I don't see the potential conflict. In this case, users could directly register for a `Registerable` getting rid of the `Registration::Item` altogether.